dist/firmware/firmware.json

```diff
@@ -83,6 +83,41 @@
                 "offset": "0x3B0000"
             }
         },
+        {
```

Comment:
While this data is correct. This file gets overwritten by the contents from the ESPFlashtool project. The version in the ESPixelStick tree is very old and no longer used.